docs/architecture/session.md

```diff
@@ -42,15 +42,16 @@ Session::create(move |ctx| {
 
 These are the messages supported by the session handlers:
 
-| Message          | Input type            | Output type                       | Description            |
-| ---------------- | --------------------- | --------------------------------- | ---------------------- |
-| `GetPeers`       | `()`                  | `()`                              | Empty                  |
+| Message          | Input type            | Output type                       | Description                            |
+| ---------------- | --------------------- | --------------------------------- | -------------------------------------- |
+| `GetPeers`       | `()`                  | `()`                              | Request peers from an outbound session |
+| `AnnounceItems`  | `Vec<InvElem>`        | `()`                              | Announce new inventory items           |
 
 #### GetPeers
-The handler of `GetPeers` message is currently empty.
-
-// TODO Update documentation when `GetPeers` gets any actual functionality.
+Request peers from an outbound session.
```

Comment:
At the moment, this is not exactly like that. Upon reception of a `GetPeers` message, the `Session` creates a `GetPeers` message and sends it through the network connection... but does not check if the session is outbound or inbound.

It's the caller (the `SessionsManager` through the `Anycast` method) the one who selects an outbound `Session` to send the `GetPeers` message to it.

However, I think that not checking the session type at the handler of the `GetPeers` in the `Session` might be a mistake. I'll create an issue for that.